test/browser/tropy_test.js

```diff
@@ -82,5 +82,48 @@ describe('Tropy', () => {
             .to.have.a.nested.property('payload.photos').eql([3])
         })
     })
+    describe('project windows', () => {
+      const projectFilePath = 'test.tpy'
+      beforeEach(() => {
+        sinon.stub(BrowserWindow.prototype, 'show')
+        tropy.state = { recents: [], win: { bounds: {} } }
+      })
+
+      afterEach(() => {
+        for (const w of tropy.wm.windows?.project  || []) {
+          w.close('project', true)
+        }
+        sinon.restore()
```

Comment:
Is there a benefit of using this over `BrowserWindow.prototype.show.restore()`? I'm a little concerned about interfering with other stubs.

Reply:
No advantage, just my lack of experienc using sinon. Will push a fix shortly